docs/production-deployment/cloud/multi-region.mdx

```diff
@@ -537,9 +537,7 @@ The `sa-east-1` region listed here is not yet available for use with multi-regio
 | `eu-west-2`      | `com.amazonaws.vpce.eu-west-2.vpce-svc-0ac7f9f07e7fb5695`      | `aws-eu-west-2.region.tmprl.cloud`      |
 | `sa-east-1`      | `com.amazonaws.vpce.sa-east-1.vpce-svc-0ca67a102f3ce525a`      | `aws-sa-east-1.region.tmprl.cloud`      |
 | `us-east-1`      | `com.amazonaws.vpce.us-east-1.vpce-svc-0822256b6575ea37f`      | `aws-us-east-1.region.tmprl.cloud`      |
-| `us-east-1`      | `com.amazonaws.vpce.us-east-1.vpce-svc-0b06bbbd81fd41222`      | `aws-us-east-1.region.tmprl.cloud`      |
 | `us-east-2`      | `com.amazonaws.vpce.us-east-2.vpce-svc-01b8dccfc6660d9d4`      | `aws-us-east-2.region.tmprl.cloud`      |
-| `us-west-2`      | `com.amazonaws.vpce.us-west-2.vpce-svc-0473f53028c65d8fb`      | `aws-us-west-2.region.tmprl.cloud`      |
 | `us-west-2`      | `com.amazonaws.vpce.us-west-2.vpce-svc-0f44b3d7302816b94`      | `aws-us-west-2.region.tmprl.cloud`      |
 
 :::tip Learn more about multi-region Namespaces
```

Comment:
I would rename the images - like image5 and 7 to more descriptive names and the details image

Reply:
Any image with a number is unused but kept in case it is needed for some reason. I will remove them instead.